components/SearchSandbox/components/RenderDataField.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
```

Comment:
Ideally, we should rename this to `DataFieldInput`

Reply:
Sure sounds good 👍